lib/iphone-simulator-xcode-simctl.ts

```diff
@@ -234,15 +234,15 @@ export class XCodeSimctlSimulator extends IPhoneSimulatorNameGetter implements I
 	}
 
 	public startSimulator(device?: IDevice): void {
-		if (!device || !device.runtimeVersion || !device.fullId) {
-			device = this.getDeviceToRun(device);
-		}
-
 		// In case the id is undefined, skip verification - we'll start default simulator.
 		if (device.id) {
```

Comment:
Check that device is not undefined `if(device && device.id)` as it is an optional parameter.